root/static/scripts/edit/externalLinks.js

```diff
@@ -727,14 +771,28 @@ const ExternalLinkRelationship =
                 </div>}
               {link.url && !link.error && !hasUrlError &&
                 <TypeDescription type={link.type} url={link.url} />}
+              {(
+                !isDateEmpty(link.begin_date) ||
+                !isDateEmpty(link.end_date) ||
+                link.ended
+              ) &&
+                <span className="date-period">
+                  {` (${formatDatePeriod(link)})`}
```

Comment:
you'll have to import bracketedText, but
```suggestion
                  {' '}
                  {bracketedText(formatDatePeriod(link))}
```